views/case-management-appointment/need-interpreter.njk

```diff
@@ -0,0 +1,58 @@
+{% set pageTitleName = i18n.pages.needInterpreterPage.pageTitle | eval %}
+{% extends "layout.njk" %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block content %}
+    {% set options = [] %}
+    {% if errorList %}
+    {{ govukErrorSummary({
+        titleText: i18n.validationErrors.errorSummary,
+        errorList: errorList
+    }) }}
+    {% endif %}
+
+    {% for item in list %}
+        {% set options = (options.push(
+            {
+                value: item.value,
+                text: item.text,
+                checked: item.checked
+            }), options) %}
+    {% endfor %}
+
+    {% call govukFieldset({
+        legend: {
+            text: i18n.pages.needInterpreterPage.title,
+            classes: "govuk-fieldset__legend--l",
+            isPageHeading: true
+        }
+    }) %}
+<form action="{{ paths.caseManagementAppointment.needInterpreter }}?_csrf={{ csrfToken }}" method="post" enctype="multipart/form-data">
```

Comment:
why this enctype? you are not uploading files here, can you please remove and move the csrf as hidden input?